packages/cli/internal/pkg/cli/workflow/manager.go

```diff
@@ -348,6 +361,37 @@ func (m *Manager) parseInputToArguments() {
 	m.arguments = []string{arguments}
 }
 
+// writeTempManifest writes the inputsFile included in the command line to the temporary MANIFEST.json located in temp directory
+// This function is only called if there is a path included in the command line with the --inputsFile flag
```

Comment:
qq: if --inputsFile flag is not used, are you saying that we are not going to create temp folder and `MANIFEST.json` file?